backend/src/nodes/pytorch_alt_nodes.py

```diff
@@ -37,14 +37,17 @@ def __init__(self):
             ),
             ImageInput(),
             ImageInput("Upscaled Background").make_optional(),
+            NumberInput(
+                label="Output Scale", default=8, minimum=1, maximum=8, unit="x"
```

Comment:
Are non-power-of-two inputs allowed?

If not, then I would suggest making the output type this:
```
let scale = Input3 & pow(2, int(0..4));
Image {
    width: scale * Input1.width,
    height: scale * Input1.height,
    channels: 3
}
```

And adding a never reason.